app/scripts/directives/bindService.js

```diff
@@ -55,20 +59,6 @@
       }
     };
 
-
-    var deploymentConfigs, deployments, replicationControllers, replicaSets, statefulSets;
-    var sortApplications = function() {
```

Comment:
Suggest leaving `sortApplications` as a function then calling your new service function from here so you don't have to repeat the parameters in every list callback.

```js
var sortApplication = function() {
  ctrl.applications = BindingModalUtils.sortApplications(deploymentConfigs, deployments, replicationControllers, replicaSets, statefulSets);
  ctrl.bindType = _.isEmpty(ctrl.applications) ? "secret-only" : "application";
};
```